Dockerfile

```diff
@@ -1,20 +1,23 @@
-ARG NODE_IMAGE=node:12.2-alpine
+ARG NODE_IMAGE=node:10.16-alpine
 ARG WORKDIR=/usr/src/app/
+ARG FLOOD_BASE_URI=/
```

Comment:
I was interested in this change myself, however the static asset files did not compile with the new URI when I build the image.

The `ARG` needs to be defined in a `FROM` block to be used with-in-it, so I suggest moving this line 3 to after line 6 as the argument is only needed in the nodebuild stage build.

So change could look like:
```Dockerfile
FROM ${NODE_IMAGE} as nodebuild
ARG WORKDIR

ARG FLOOD_BASE_URI=/
ENV FLOOD_BASE_URI $FLOOD_BASE_URI
```

[Docker docs](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact):
> An ARG declared before a FROM is outside of a build stage, so it can’t be used in any instruction after a FROM. To use the default value of an ARG declared before the first FROM use an ARG instruction without a value inside of a build stage.